README.md

```diff
@@ -54,7 +54,7 @@ export INSIGHTS_CONTROLLER_CONFIG_FILE=~/config.toml
 ## Data storage
 
 Data storage used by the service is configurable via the command line parameters. Currently it is possible to configure the following data storages:
-* SQLite local database: `controller.db` for the local deployment and `data.db` for functional tests
+* SQLite local database: `controller.db` for the local deployment and `test.db` for functional tests
```

Comment:
Not so sure about this one, but it looks like only `test.db` file is used in the tests.